buffer/src/main/java/io/netty/buffer/PoolArenaMetric.java

```diff
@@ -21,18 +21,13 @@
 /**
  * Expose metrics for an arena.
  */
-public interface PoolArenaMetric {
+public interface PoolArenaMetric extends SizeClassesMetric {
 
     /**
      * Returns the number of thread caches backed by this arena.
      */
     int numThreadCaches();
 
-    /**
-     * Returns the number of tiny sub-pages for the arena.
-     */
-    int numTinySubpages();
```

Comment:
Need to keep these public methods to maintain binary compatibility, implementations could return 0, empty collection etc. and mark them deprecated.

Reply:
Done.